ustriage/ustriage.py

```diff
@@ -382,4 +386,7 @@ def launch():
 
 
 if __name__ == '__main__':
+    from task import Task
     launch()
+else:
+    from .task import Task
```

Comment:
Import should be at the top and only `from .task import Task` should be required. Did you find otherwise?

Reply:
Yeah, with this at the top local execution wasn't possible at all for me.
See commit 21e754c16f95793bb54a38ee9d3c76e0024362e1

I understood your old commit that made it absolute, but that would not want to work

```
$ ./ustriage.py 
Traceback (most recent call last):
  File "./ustriage.py", line 23, in <module>
    from .task import Task
ModuleNotFoundError: No module named '__main__.task'; '__main__' is not a package
```

I experimented with different local dirs for the rel paths or using it as a module and such (-m), but none worked for me. Changing back the import was the only way to get it running.

I did not want to revert your old change either, so I came up with something that works following most of the discussions around similar issues I found. But thinking more about it I think in our case execution from the snap would hit my new import as well, so you'd be back to the old problem you fixed with 3995e5e729674e8cbdeb7b2ebbb41f5ab994d882

Maybe your python-foo is better and you find/can-suggest something working for both local-from-git and snap?

Reply:
odd, this is the diff I used:
https://paste.ubuntu.com/p/GrjkV45rm6/

and it ran fine on the CLI and the built snap:
https://paste.ubuntu.com/p/YmRmvg8bDn/

Did you try something similar to that?

Reply:
Yeah that I tried and it didn't work.
But now it does, odd - whatever it was we still fixed the Readme about local execution with that then.
I'll wrap your suggestion into my fix and push - then the PR should be overall fine.